acceleration/fast_inference_tutorial/fast_inference_tutorial.ipynb

```diff
@@ -0,0 +1,646 @@
+    "        model = SegResNet(\n",
+    "            spatial_dims=3,\n",
+    "            in_channels=1,\n",
+    "            out_channels=105,\n",
+    "            init_filters=32,\n",
+    "            blocks_down=[1, 2, 2, 4],\n",
+    "            blocks_up=[1, 1, 1],\n",
+    "            dropout_prob=0.2,\n",
+    "        )\n",
+    "        weights = torch.load(weights_path)\n",
+    "        model.load_state_dict(weights)\n",
+    "        model.to(device)\n",
+    "        model.eval()\n",
+    "    else:\n",
+    "        model = torch.jit.load(trt_model_path)\n",
+    "    return model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define Inference Workflow\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def run_inference(data_list, infer_transforms, model, device, benchmark_type):\n",
+    "    total_time_dict = {}\n",
+    "    roi_size = (96, 96, 96)\n",
+    "    sw_batch_size = 4\n",
+    "\n",
+    "    for idx, sample in enumerate(data_list[:10]):\n",
+    "        start = timer()\n",
+    "        data = infer_transforms({\"image\": sample})\n",
+    "\n",
+    "        with torch.no_grad():\n",
+    "            input_image = (\n",
+    "                data[\"image\"].unsqueeze(0).to(device)\n",
+    "                if benchmark_type in [\"trt\", \"original\"]\n",
+    "                else data[\"image\"].unsqueeze(0)\n",
+    "            )\n",
+    "\n",
+    "            output_image = sliding_window_inference(input_image, roi_size, sw_batch_size, model)\n",
+    "            output_image = output_image.cpu()\n",
+    "\n",
+    "            end = timer()\n",
+    "\n",
+    "        print(output_image.mean())\n",
+    "\n",
+    "        del data\n",
+    "        del input_image\n",
+    "        del output_image\n",
+    "        torch.cuda.empty_cache()\n",
+    "        gc.collect()\n",
+    "\n",
+    "        sample_name = sample.split(\"/\")[-1]\n",
+    "        if idx > 0:\n",
+    "            total_time_dict[sample_name] = end - start\n",
+    "            print(end - start)\n",
+    "    return total_time_dict"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Running the Benchmark\n",
+    "\n",
+    "The cell below will execute the benchmark based on the `benchmark_type` variable.\n",
+    "\n",
+    "#### (Optional) Using the Python Script\n",
+    "\n",
+    "For convenience, a Python script, [`run_benchmark.py`](./run_benchmark.py), is available to run the benchmark. You can open a terminal and execute the following command to run the benchmark for all benchmark types:\n",
+    "\n",
+    "\n",
+    "```bash\n",
+    "for benchmark_type in \"original\" \"trt\" \"trt_gpu_transforms\" \"trt_gds_gpu_transforms\"; do\n",
+    "    python run_benchmark.py --benchmark_type \"$benchmark_type\"\n",
+    "done\n",
```

Comment:
You could instead put this into a cell with `%%bash` at the top to allow users to run command, or you could do it with Python more directly for those that don't have bash:

```python
for benchmark_type in ("original", "trt", "trt_gpu_transforms", "trt_gds_gpu_transforms"):
    !python run_benchmark.py --benchmark_type {benchmark_type}
```

Reply:
You should also state here that the script contains the same code as what's in this notebook, running it will generate a csv with the results for each type, but if the user wants to run the benchmark here in this notebook then can run the following cell with the commented lines uncommented.